### PR TITLE
Cleanup whitespace

### PR DESCRIPTION
**Pull request by ggravlingen**
#### Reason for change
This is a follow-up to https://github.com/Arelle/Arelle/pull/276#pullrequestreview-1072850392.

#### Description of change
I have run the following script to cleanup all unnecessary whitespace:
```
import os

PATH = '/workspaces/Arelle/arelle'

for path, dirs, files in os.walk(PATH):
    for f in files:
        file_name, file_extension = os.path.splitext(f)
        if file_extension == '.py':
            path_name = os.path.join(path, f)
            with open(path_name, 'r') as fh:
                new = [line.rstrip() for line in fh]
            with open(path_name, 'w') as fh:
                [fh.write('%s\n' % line) for line in new]

```

#### Steps to Test

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
The items listed below may not capture all security relevant changes. Before providing a security review, be sure to review the entire PR for security impact.
<details>
<summary>(3) Security relevant changes were detected</summary>
<li>Watched keyword <code>hashlib</code> in <code>arelle/plugin/internet/proxyNTLM/ntlm.py</code> line(s) ['21', '375', '377', '411', '422'] added</li>
<li>Watched keyword <code>hashlib</code> in <code>arelle/plugin/internet/proxyNTLM/ntlm.py</code> line(s) ['21', '375', '377', '411', '422'] removed</li>
<li>Watched keyword <code>XMLParser(</code> in <code>arelle/plugin/formulaLoader.py</code> line(s) ['1934'] added</li>
</details>

## Action Items
* Request infosec review via Slack: #support-infosec
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Comment by ggravlingen** [comment]
Regarding the massive change to the git history, maybe this link that I brought up when we discussed running Black is relevant here too: https://black.readthedocs.io/en/stable/guides/introducing_black_to_your_project.html

**Review by sagesmith-wf — APPROVED** [review]
Congrats on your number 2 contributor status after this lol

**Comment by ggravlingen** [comment]
😅😅😅

**Comment by austinmatherne-wk** [comment]
> Regarding the massive change to the git history, maybe this link that I brought up when we discussed running Black is relevant here too: [black.readthedocs.io/en/stable/guides/introducing_black_to_your_project.html](https://black.readthedocs.io/en/stable/guides/introducing_black_to_your_project.html)

Yes, please. Add that as part of this PR:
```
# .git-blame-ignore-revs
# Removed trailing whitespace from the entire codebase
956cfcca1aaa18dd2521172a851da3375abd2978
```

**Comment by ggravlingen** [comment]
> > Regarding the massive change to the git history, maybe this link that I brought up when we discussed running Black is relevant here too: [black.readthedocs.io/en/stable/guides/introducing_black_to_your_project.html](https://black.readthedocs.io/en/stable/guides/introducing_black_to_your_project.html)
> 
> Yes, please. Add that as part of this PR:
> 
> ```
> # .git-blame-ignore-revs
> # Removed trailing whitespace from the entire codebase
> 956cfcca1aaa18dd2521172a851da3375abd2978
> ```

Did you mean like so?

**Review by sagesmith-wf — APPROVED** [review]

File: arelle/plugin/xbrlDB/SqlDb.py
Comment (austinmatherne-wk) [inline arelle/plugin/xbrlDB/SqlDb.py:681]: Calling this out as an example of trailing whitespace that does cause a difference at runtime. This example is inconsequential, but whoever QAs this needs to go through the diffs to check for others.

**Review by austinmatherne-wk — APPROVED** [review]

File: arelle/plugin/xbrlDB/SqlDb.py
Comment (ggravlingen) [reply arelle/plugin/xbrlDB/SqlDb.py:681]: Should we maybe break this down into smaller chunks, eg all files a-h and so on in separate PRs? Might make it a bit easier to review?

Comment (austinmatherne-wk) [reply arelle/plugin/xbrlDB/SqlDb.py:681]: This is okay. I prefer it as a single commit. I'll review/QA, I just need a chunk of time to go through it.

Comment (ggravlingen) [reply arelle/plugin/xbrlDB/SqlDb.py:681]: OK, thanks!

Comment (austinmatherne-wk) [reply arelle/plugin/xbrlDB/SqlDb.py:681]: I reviewed and didn't find any problematic changes. I can merge once the merge conflicts are resolved, just don't rebase this one, that would make verifying again more difficult in this case.

Comment (ggravlingen) [reply arelle/plugin/xbrlDB/SqlDb.py:681]: I resolved conflicts from within GitHub here on the web and just noticed the merge message was merge master into this branch. I think that’s specifically what you didn’t want. Really sorry about this mess up. 🤦‍♂️

**Review by austinmatherne-wk — APPROVED** [review]

File: arelle/plugin/xbrlDB/SqlDb.py
Comment (austinmatherne-wk) [reply arelle/plugin/xbrlDB/SqlDb.py:681]: That's actually what I wanted. 🙂 The merge commit was easy to review. After the CI run passes I'll merge this.

Comment (ggravlingen) [reply arelle/plugin/xbrlDB/SqlDb.py:681]: 😮‍💨 thanks for reviewing this!

**Pull request merged**
